### DISCUSSION TITLE
Git hub not compatible with my PC

### DISCUSSION THREAD

**Discussion by Paulagll** [community/community#57357] [Programming Help]
I just downladed Git bash ans now its saying the app cannot run on my pc and is saying to reach out to the software publisher. what should i do?

**Comment by queenofcorgis** [comment]
Hi @Paulagll ,

GitHub is not the publisher of Git Bash, I've moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
